app/components/record_actions/view.rb

```diff
@@ -45,5 +55,17 @@ def withdraw_link
     def reinstate_link
       govuk_link_to(t("views.trainees.edit.reinstate"), trainee_reinstatement_path(trainee), class: "reinstate")
     end
+
+    def trainee_commencement_date_in_future?
+      trainee.commencement_date && trainee.commencement_date > Time.zone.now
+    end
+
+    def course_started_but_trainee_has_not_specified_start_date?
+      course_start_date && course_start_date < Time.zone.now && trainee.commencement_date.blank?
+    end
+
+    def course_start_date
+      trainee.published_course&.start_date
```

Comment:
aren't we supposed to be using the `trainee.course_start_date` as it's copied over from the course? It could be different from the original course associated with this trainee